template.yaml

```diff
@@ -250,15 +250,17 @@ Resources:
                   Action:
                     - ssm:SendCommand
                     - ssm:GetCommandInvocation
-                  Resource: '*'
+                  Resource:
+                    - !Sub "arn:aws:ssm:${AWS::Region}::document/AWS-RunShellScript"
```

Comment:
This will work for the `ssm:SendCommand` action. However, it is not possible to limit the `ssm:GetCommandInvocation` action by resource¹.

To get the resource restricction for `ssm:SendCommand`, you separate the 2 actions into different statements.

---

¹ In the [IAM Actions table for SSM](https://docs.aws.amazon.com/service-authorization/latest/reference/list_awssystemsmanager.html), there are no entries in the "Resource types" column for `GetCommandInvocation`. This means we unfortunately have to set `Resource: "*"`.

Reply:
Missed that one! Addressed now. Thanks, @lucaswilric!